<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_File_1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>choose your file#1</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_File_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>104</width>
     <height>21</height>
    </rect>
   </property>
   <property name="plainText">
    <string>your file#1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_run">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>340</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>RUN</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>104</width>
     <height>21</height>
    </rect>
   </property>
   <property name="plainText">
    <string>your file#2
</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_File_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_File_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>choose your file#2</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_File_1">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>110</y>
     <width>104</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_File_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>240</y>
     <width>104</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>30</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
